apps/andi/mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Andi.MixProject do
   def project do
     [
       app: :andi,
-      version: "0.14.4",
+      version: "0.15.0",
```

Comment:
You'll need to bump this again (sorry).  We also used version 0.15.0 for our change to bring in changes to smart_city.